### ISSUE TITLE
Establish a consistent naming scheme of global objects

### ISSUE THREAD

**Issue by asbjornu** [PayEx/design.payex.com#14]
The general Design Guide scripts are located in the object `px.script`. I think the word `script` here makes no sense. Why not call it `PayEx.DesignGuide`? We obviously need to establish a global standard around JavaScript namespacing in PayEx, as [Checkout v2](https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/payex-checkout/) is using `payex.hostedView` for instance.

Creating all of these global, inconsistently named objects is confusing and looks unprofessional. I think it should be down to the Design Guide to establish how these global objects are named and in that case eCom might have made the choice for us in its rollout of `payex.hostedView`.

I would prefer pascal casing for "classes" as this, but consistency is more important than my personal preference. I therefore suggest the Design Guide groups all of its globals into `payex.designGuide.*`.

**Comment by erikhallaraaker** [comment]
I agree that we need a global standard, but `Payex.DesignGuide.Alert.init()` seems unnecessary long though..

**Comment by hmosvoll** [comment]
Do we need init, can't alert just be a function?

Something like `payex.designGuide.alert()`?

**Comment by asbjornu** [comment]
Also, do we need init at all? If Bootstrap can do without init, shouldn't we too?

**Comment by asbjornu** [comment]
I registered #16 for making Design Guide do `.init()` itself so all of these functions hopefully can be deleted. The point about having a consistent naming standard still stands, though.

**Comment by erikhallaraaker** [comment]
The problem is that some of the components have utility functions, and per the issue, the goal is to keep it consistent.

We don't NEED the init functions, they are there to give users more control over the components should they wish to do so.

**Comment by asbjornu** [comment]
That makes sense, but as per #16, we should make them optional and per this issue we should ensure a consistent, sensible naming convention. I like @hmosvoll's suggestion of just using the name of the component as an initializer/configurator function, though.

**Comment by erikhallaraaker** [comment]
Yeah, I agree with using the name as the initializer, the problem is when a component name contains utility functions as well.
Also, they are already optional for regular routing, see my comment on #16

**Comment by hmosvoll** [comment]
Chaining?
`payex.designGuide.alert()` could return an object with the util functions, so it could be used like this:

```
payex.designGuide.alert().utilFunction();

// or

var alert = payex.designGuide.alert();
alert.utilFunction();
```
